src/InputWithLabel.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+const InputWithLabel = (props) => {
+    const inputRef = React.useRef();
+
+    React.useEffect(() => {
+            inputRef.current.focus();
+    });
+
```

Comment:
Try to figure out how to use the isFocused prop in this method. The road to react book has a good example of it.